docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -123,6 +123,7 @@ The tuningConfig is optional and default parameters will be used if no tuningCon
 |`buildV9Directly`|Boolean|Whether to build a v9 index directly instead of first building a v8 index and then converting it to v9 format.|no (default == false)|
 |`reportParseExceptions`|Boolean|If true, exceptions encountered during parsing will be thrown and will halt ingestion; if false, unparseable rows and fields will be skipped.|no (default == false)|
 |`handoffConditionTimeout`|Long|Milliseconds to wait for segment handoff. It must be >= 0, where 0 means to wait forever.|no (default == 0)|
+|`resetOffsetAutomatically`|Boolean|Whether to reset the consumer offset if the next offset that it is trying to fetch is less than the earliest available offset for that particular partition. The consumer offset will be reset to either the earliest or latest offset depending on `useEarliestOffset` property of `KafkaSupervisorIOConfig` (see below). WARNING - Getting into this situation generally leads to data loss, therefore setting this to `true` means tasks will skip events from kafka without halting. If set to false then ingestion for that particular partition will halt and manual intervention is required to correct the situation, please see `Reset Supervisor` API below.|no (default == false)|
```

Comment:
Instead of saying 'leads to data loss' which could be misconstrued to mean they'll lose existing data in Druid, maybe phrase it something more like 'this situation typically occurs when messages in Kafka are no longer available for consumption and therefore won't be ingested into Druid'